code/controllers/subsystems/processing/organs/internal_wounds.dm

```diff
@@ -0,0 +1,7 @@
+PROCESSING_SUBSYSTEM_DEF(internal_wounds)
+	name = "Internal Wounds"
+	priority = 101 // Fires before mob Life()
```

Comment:
Consider turning it into a define, so it's in line with other subsystems.

Reply:
Done. Changed it to 99 so it fires after mob Life(), as well.